source/parameters/bend.md

```diff
@@ -69,9 +68,7 @@ The values of `e1` and `e2` give the rotation angle of the entrance and exit pol
 respectively with respect to the radial {math}`x_1` and {math}`x_2` axes as shown in {numref}`f:bend`.
 Zero `e1` and `e2` gives a wedge shaped magnet.
 Also see `e1_rect` and `e2_rect`. The relationship is
-  ```{code} python
-  :label: eeaeea
-
+  ```{code} yaml
```

Comment:
Should this be rendered in math mode?
```suggestion
  ```{code} math
```

Reply:
I like yaml, my suggestion is to add a self-consistent example to each element that lists all properties as a valid yaml snippet.

We can add `# comments` on each property to describe if it is optional and what its implicit defaults are if skipped.